hystrix-contrib/hystrix-javanica/hystrix-javanica-aspectj/src/main/java/com/netflix/hystrix/contrib/javanica/aop/aspectj/MethodExecutionAction.java

```diff
@@ -39,17 +41,17 @@
     private final Object object;
     private final Method method;
     private final Object[] _args;
-    private final MetaHolder metaHolder;
+    private final AspectjMetaHolder metaHolder;
```

Comment:
Basically, they have nothing in common and ideally this is something each implementation(aspectj, aop-alliance etc) should define I think. I feel we will not gain much by providing an abstraction here. Anyway, let me think fresh on this tomorrow morning:)